salt/modules/keystoneng.py

```diff
@@ -91,8 +91,8 @@ def _clean_kwargs(keep_name=False, **kwargs):
         kwargs['name_or_id'] = kwargs.pop('name')
 
     try:
-        clean_func = salt.utils.args.clean_kwargs
-    except AttributeError:
+        clean_func = __utils__['args.clean_kwargs']
+    except KeyError:
         clean_func = salt.utils.clean_kwargs
     return clean_func(**kwargs)
```

Comment:
Please replace this entire block with either

```python
return __utils__['args.clean_kwargs'](**kwargs)
```
or
```python
return salt.utils.args.clean_kwargs(**kwargs)
```

The develop branch has deprecated the use of `salt.utils` and it will generate a deprecation warning if used.